### DISCUSSION TITLE
This pop-up window appears every time I open the app. It’s so annoying. Why don’t other apps have this problem?

### DISCUSSION THREAD

**Discussion by kjxbyz** [community/community#65476] [Mobile]
### Select Topic Area

Bug

### Body

This pop-up window appears every time I open the app. It’s so annoying. Why don’t other apps have this problem?

![IMG_1603](https://github.com/community/community/assets/47768002/61927f35-75ef-4fa6-89db-18ffdc389536)

**Comment by alcere** [comment]
Hi @kjxbyz! That error message looks like unrelated to the GitHub app, could it be related to new privacy features introduced by iOS 16?
